nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/java/org/apache/nifi/processors/mongodb/GetMongo.java

```diff
@@ -99,6 +100,12 @@ public ValidationResult validate(final String subject, final String value, final
         .required(false)
         .addValidator(StandardValidators.POSITIVE_INTEGER_VALIDATOR)
         .build();
+    static final PropertyDescriptor RESULTS_PER_FLOWFILE = new PropertyDescriptor.Builder()
+        .name("Results Per FlowFile")
```

Comment:
Could you use .displayName() for what will be displayed in the UI for the property name? and .name() is a more "computer-friendly" name. It's not mandatory but it's a convention we're trying to adopt.